playbooks/roles/common_vars/defaults/main.yml

```diff
@@ -14,13 +14,17 @@ COMMON_BASIC_AUTH_EXCEPTIONS:
 # Settings to use for calls to edxapp manage.py
 COMMON_EDXAPP_SETTINGS: 'aws'
 
+# Set to True to install aws role when running edx_sandbox playbook.
+COMMON_ENABLE_AWS_INTEGRATION: False
+
 # Turn on syncing logs on rotation for edx
 # application and tracking logs, must also
 # have the aws or openstack role installed
 COMMON_OBJECT_STORE_LOG_SYNC: False
 COMMON_OBJECT_STORE_LOG_SYNC_BUCKET: "edx-{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT }}"
 COMMON_OBJECT_STORE_LOG_SYNC_PREFIX: "logs/tracking/"
 COMMON_OBJECT_STORE_LOG_SYNC_SCRIPT: "{{ COMMON_BIN_DIR }}/send-logs-to-object-store"
+COMMON_OBJECT_STORE_LOG_SYNC_ON_EXIT: "{{ COMMON_BIN_DIR }}/sync-logs-on-exit"
```

Comment:
@lgp171188, that's correct, but it would also be confusing to change it in this configuration when this version has already been [merged](https://github.com/open-craft/configuration/pull/56) and cherry-picked to 6 different branches.

Reply:
@Agrendalath, this one's on me. I had created this review comment long back but forgot to submit the review :-( So let it be like this. We can fix this later.